lnwire/announcement_signatures.go

```diff
@@ -75,19 +104,5 @@ func (a *AnnounceSignatures) MsgType() MessageType {
 //
 // This is part of the lnwire.Message interface.
 func (a *AnnounceSignatures) MaxPayloadLength(pver uint32) uint32 {
-	var length uint32
-
-	// ChannelID - 36 bytes
-	length += 36
-
-	// ShortChannelID - 8 bytes
-	length += 8
-
-	// NodeSignatures - 64 bytes
-	length += 64
-
-	// BitcoinSignatures - 64 bytes
-	length += 64
-
-	return length
+	return 65533
```

Comment:
Since we never check this data in any way, I think this means nodes can start sending us large messages (up to `MaxPayloadLength`) that we will happily store and forward. Should we instead set a soft limit on the max msg length we accept, with a reasonable buffer length for future added fields?

Reply:
Yep for these classes of messages that we'll store. We can add a limiter later if/when it proves to be an issue, to my knowledge there' nothing on mainnet atm that's excessively padded. The other day I started to tool to traverse select buckets to compute the current disk utilization, something that we should start monitoring given the recent chan update index bug, and the fact that bolt by default won't automatically reclaim space.